todoapp/app/proguard-rules.pro

```diff
@@ -1,10 +1,18 @@
+-dontoptimize
+
 # Some methods are only called from tests, so make sure the shrinker keeps them.
 -keep class com.example.android.architecture.blueprints.** { *; }
 
--keep class android.support.v4.widget.DrawerLayout { *; }
--keep class android.support.test.espresso.IdlingResource { *; }
--keep class android.support.test.espresso.IdlingRegistry { *; }
+-keep class androidx.drawerlayout.widget.DrawerLayout { *; }
```

Comment:
This shouldn't be needed as AAPT emits keep rules for Views?!

Reply:
java.lang.NoSuchMethodError: No virtual method isDrawerVisible(I)Z in class Landroidx/drawerlayout/widget/DrawerLayout; or its super classes (declaration of 'androidx.drawerlayout.widget.DrawerLayout' appears in /data/app/com.example.android.architecture.blueprints.todomvp.mock-0ZkU6c_Gfa12K-fi5IOT1A==/base.apk)